### PR TITLE
add rule elements to remaining Player Core Spellshape feats

### PR DESCRIPTION
**Pull request by ammalagonc**
I updated the re-en.json spellshape localization text for some feats and added rule elements to the following spellshape feats:

- CastDown
- Martyr
- HeroicRecovery
- DefensiveRecovery
- EbbAndFlow
- Remediate
- EchoingChannel
- WidenSpell
- FormControl
- SnowdriftSpell
- CurrentSpell
- HealingTransformation
- ThunderclapSpell
- GarlandSpell
- SowSpell
- LeyLineConduit
- SplitHex

### CODE REVIEW

**Comment by simonward** [comment]
It looks like the description for "Current Spell (Elementalist)" links to the old effects.

**Comment by ammalagonc** [comment]
Ooh
Good point
Let me correct that

**Comment by TikaelSol** [comment]
You've got some merge conflicts

**Comment by ammalagonc** [comment]
Merge conflicts should be resolved now.

File: packs/feat-effects/effect-current-spell.json [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+                "key": "FlatModifier",
+                "label": "Effect: Current Spell against ranged attacks",
+                "predicate": [
+                    "ranged"
+                ],
+                "selector": "ac",
+                "slug": "effect-current-spell-ranged",
+                "type": "circumstance",
+                "value": 2
+            }
```
Comment (TikaelSol) [inline packs/feat-effects/effect-current-spell.json:66]:
```suggestion
                "key": "FlatModifier",
                "predicate": [
                    "{item|flags.pf2e.rulesSelections.element}"
                ],
                "selector": "saving-throw",
                "type": "circumstance",
                "value": 1
            },
            {
                "key": "FlatModifier",
                "predicate": [
                    "ranged"
                ],
                "selector": "ac",
                "slug": "effect-current-spell-ranged",
                "type": "circumstance",
                "value": 2
            }
```

File: packs/feat-effects/effect-current-spell.json [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+                        "label": "PF2E.TraitWater",
+                        "value": "water"
+                    },
+                    {
+                        "label": "PF2E.TraitAir",
+                        "value": "air"
+                    }
+                ],
+                "flag": "element",
+                "key": "ChoiceSet"
```
Comment (TikaelSol) [inline packs/feat-effects/effect-current-spell.json:36]:
```suggestion
                "key": "ChoiceSet",
                "prompt": "PF2E.SpecificRule.Prompt.Element"
```

File: packs/feat-effects/effect-current-spell.json [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+            },
+            {
+                "key": "FlatModifier",
+                "label": "Effect: Current Spell",
+                "mode": "add",
+                "selector": "ac",
```
Comment (TikaelSol) [inline packs/feat-effects/effect-current-spell.json:42]:
```suggestion
                "key": "FlatModifier",
                "selector": "ac",
```

File: packs/feat-effects/effect-current-spell.json [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+                "label": "Against effects with the selected trait",
+                "predicate": [
+                    "{item|flags.pf2e.rulesSelections.element}"
+                ],
+                "selector": "saving-throw",
+                "type": "circumstance",
+                "value": 1
+            },
+            {
+                "key": "FlatModifier",
```
Comment (TikaelSol) [inline packs/feat-effects/effect-current-spell.json:57]: Use adjust modifier instead

File: packs/feats/reptile-rider.json [hunk 5]
```diff
@@ -11,7 +11,7 @@
...
+            "value": "<p>To you, crocodilians, giant lizards, and dinosaurs are loyal steeds, not monsters. You gain the @UUID[Compendium.pf2e.feats-srd.Item.Ride] feat, even if you don't meet the prerequisites. You gain a +1 circumstance bonus to Nature checks to Handle an Animal as long as the animal is a reptile, dinosaur, or even a non-sapient dragon.</p>\n<hr />\n<p><em><strong>Note</strong> This feat refers to Nature checks to Handle an Animal. This check does not exist in Pathfinder 2, and so the feat instead gives a bonus to Nature checks to @UUID[Compendium.pf2e.actionspf2e.Item.Command an Animal].</em></p>"
```
Comment (TikaelSol) [inline packs/feats/reptile-rider.json:14]:
```suggestion
            "value": "<p>To you, crocodilians, giant lizards, and dinosaurs are loyal steeds, not monsters. You gain the @UUID[Compendium.pf2e.feats-srd.Item.Ride] feat, even if you don't meet the prerequisites. You gain a +1 circumstance bonus to Nature checks to @UUID[Compendium.pf2e.actionspf2e.Item.Command an Animal] as long as the animal is a reptile, dinosaur, or even a non-sapient dragon.</p>"
```
I'm fine fixing the obvious typo.

**Pull request merged**